### PR TITLE
Add python version's mobilenet

### PR DESCRIPTION
**Pull request by rickycao-qy**
This PR includes:
1. Move download source of pre-trained weights to OSS.
2. Add python version's mobilenet

### CODE REVIEW

File: packages/costa/src/runtime.ts [hunk 1]
```diff
@@ -131,8 +131,10 @@ export class CostaRuntime {
       pkg = selectNpmPackage(meta, source);
     } else if (source.from === 'git') {
       debug(`requesting the url ${source.uri}...`);
-      const { hostname, pathname } = source.urlObject;
-      pkg = await fetchPackageJsonFromGit(`git@${hostname}:${pathname}`, 'HEAD');
+      const { hostname, auth, hash } = source.urlObject;
+      let pathname = source.urlObject.pathname.replace(/^\/:?/, '');
+      const remote = `${auth || 'git'}@${hostname}:${pathname}${hash || ''}`;
+      pkg = await fetchPackageJsonFromGit(remote, 'HEAD');
```
Comment (yorkie) [inline packages/costa/src/runtime.ts:137]: This is merged by #238, you can rebase from `master`.

File: packages/plugins/model-define/tensorflow-mobilenet-model-define/package.json [hunk 2]
```diff
@@ -14,23 +14,29 @@
...
-    "@tensorflow/tfjs-node-gpu": "1.7.0",
-    "jimp": "^0.10.0"
+    "fs-extra": "^8.1.0"
   },
   "gitHead": "1e21ad0799e221dcf4b7aa3a3b2187d5b8d8b865",
   "devDependencies": {
     "@types/jasmine": "^3.5.7",
-    "nyc": "14.1.1"
+    "nyc": "14.1.1",
+    "@types/fs-extra": "^8.1.0"
```
Comment (yorkie) [inline packages/plugins/model-define/tensorflow-mobilenet-model-define/package.json:27]: Reorder it in the alphabet

File: packages/costa/src/runtime.ts (hunk 1 above)
Comment (rickycao-qy) [reply packages/costa/src/runtime.ts]: has rebased

File: packages/plugins/model-define/tensorflow-mobilenet-model-define/package.json (hunk 2 above)
Comment (rickycao-qy) [reply packages/plugins/model-define/tensorflow-mobilenet-model-define/package.json]: fixed

File: packages/plugins/model-define/detectron-maskrcnn-model-define/src/index.ts [hunk 3]
```diff
@@ -1,13 +1,28 @@
...
+const MODEL_URL =
+  `http://ai-sample.oss-cn-hangzhou.aliyuncs.com/pipcook/models/detectron_r50/${MODEL_WEIGHTS_NAME}`;
+const MODEL_PATH = path.join(
+  os.homedir(),
+  '.torch',
+  'fvcore_cache',
+  'detectron2',
+  'ImageNetPretrained',
+  'MSRA',
+  MODEL_WEIGHTS_NAME
```
Comment (yorkie) [inline packages/plugins/model-define/detectron-maskrcnn-model-define/src/index.ts:18]: Ditto

File: packages/plugins/model-define/tensorflow-resnet-model-define/src/index.ts [hunk 4]
```diff
@@ -15,6 +16,11 @@ const { ResNet50 } = boa.import('tensorflow.keras.applications.resnet50');
 const { GlobalAveragePooling2D, Dropout, Dense } = boa.import('tensorflow.keras.layers');
 const { Model } = boa.import('tensorflow.keras.models');
 
+const MODEL_WEIGHTS_NAME = 'resnet50_weights_tf_dim_ordering_tf_kernels_notop.h5';
+const MODEL_URL =
+  `http://ai-sample.oss-cn-hangzhou.aliyuncs.com/pipcook/models/resnet50_python/${MODEL_WEIGHTS_NAME}`;
+const MODEL_PATH = path.join(os.homedir(), '.keras', 'models', MODEL_WEIGHTS_NAME);
```
Comment (yorkie) [inline packages/plugins/model-define/tensorflow-resnet-model-define/src/index.ts:22]: Ditto

Comment (rickycao-qy) [reply packages/plugins/model-define/tensorflow-resnet-model-define/src/index.ts]: fixed

File: packages/plugins/model-define/detectron-maskrcnn-model-define/src/index.ts (hunk 3 above)
Comment (rickycao-qy) [reply packages/plugins/model-define/detectron-maskrcnn-model-define/src/index.ts:17]: fixed

File: packages/plugins/model-define/detectron-fasterrcnn-model-define/src/index.ts [hunk 5]
```diff
@@ -1,13 +1,27 @@
...
+const MODEL_URL =
+  `http://ai-sample.oss-cn-hangzhou.aliyuncs.com/pipcook/models/detectron_r50/${MODEL_WEIGHTS_NAME}`;
+const MODEL_PATH = path.join(
+  os.homedir(),
+  '.torch',
+  'fvcore_cache',
+  'detectron2',
+  'ImageNetPretrained',
+  'MSRA',
+  MODEL_WEIGHTS_NAME
```
[the remark this answers is not in the document]
Comment (rickycao-qy) [reply packages/plugins/model-define/detectron-fasterrcnn-model-define/src/index.ts:17]: fixed

**Pull request merged**